src/components/blocks/ovc/footerOVC.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
```

Comment:
Scott, forgive me! I just looked at my own code and 've been naming all my site-specific components with dashes. 🤣 You can totally switch it back to footer-ovc.js if you want.